core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/configuration/action/ActionConfiguration.java

```diff
@@ -177,7 +179,8 @@ private String deleteStatement(ActionKey actionKey) {
         List<Action> actions = new ArrayList<>();
         String query = "select * from " + getMetadataRepository().getTableNameByLabel("Actions") +
                 " where SCRIPT_ID = " + SQLTools.getStringForSQL(scriptKey.getScriptId()) +
-                " and SCRIPT_VRS_NB = " + SQLTools.getStringForSQL(scriptKey.getScriptVersion())
+                " and SCRIPT_VRS_NB = " + SQLTools.getStringForSQL(scriptKey.getScriptVersion()) +
+                " and DELETED_AT = 'NA' "
```

Comment:
doesn't ActionKey contain deletedAt? don't we need to use this value?

Reply:
It is present in ScriptKey, but we cant pass the exact value otherwise it will also return deleted Action.